gulpfile.js

```diff
@@ -133,13 +130,6 @@ export async function buildTranslations() {
 
 export const buildTestApp = gulp.parallel(buildTestAppCss, buildTranslations, buildTestAppJs);
 
-export async function countriesUpdate() {
-  await Promise.all(supportedLanguages.map(lang => downloadJson(
-    `https://raw.githubusercontent.com/umpirsky/country-list/master/data/${encodeURIComponent(lang)}/country.json`,
-    `./src/data/country-names/${lang}.json`
-  )));
-}
-
```

Comment:
The countries were only used in the old user profile that we replaced with the new user page.